src/site/content/en/learn/html/audio-video/index.md

```diff
@@ -3,8 +3,203 @@ title: 'Audio and Video'
+doesn't load, this content is shown. The closing `</video>` tag is required, even if there is no content between the two
+(but there should always be fallback content , right?).
+
+If no `src` attribute is included on the opening `<video>` or `<audio>` tags, include one or more [`<source>`](https://developer.mozilla.org/docs/Web/HTML/Element/source) elements,
+each with a `src` attribute to a media file. The following example includes three media file options, fallback content,
+and English and French subtitles between the opening and closing tags.
+
+```html
+<video controls poster="images/machine.jpg">
+   <source src="videos/machines.webm" type="video/webm">
+   <source src="videos/machines.mp4" type="video/mp4">
+   <source src="videos/machines.ogv" type="video/ogg">
+   <track label="English" kind="subtitles" srclang="en" src="vtt/subtitles-en.vtt" default />
+   <track label="Francais" kind="subtitles" srclang="fr" src="vtt/subtitles-fr.vtt" />
+   <p>Watch <a href="https://youtube.com/link">video on Youtube</a>
+</video>
+```
+
+Each `<source>` child includes a `src` attribute pointing to the resource and the `type` attribute informs the browser
+of the linked file's [media type](https://developer.mozilla.org/docs/Web/Media/Formats/Containers). This prevents the browser
+from fetching media files it wouldn't be able to decode.
+
+Within the `type` attribute, you can include a [`codecs`](https://developer.mozilla.org/docs/Web/Media/Formats/codecs_parameter) parameter,
+which specifies exactly how the resource is encoded. Codecs give you a way of including media optimizations that are not yet supported
+in all browsers. The codec is separated from the media type with a semicolon. For example, the codec can be written using
+intuitive syntax, such as `<source src="videos/machines.webm" type="video/webm;codecs=vp8,vorbis">` which indicates that the
+WebM files contain VP8 video and vorbis audio. Codecs can also be more difficult to decipher, such as `<source src="videos/machines.mp4" type="video/mp4"; codecs=avc1.4d002a">`
+which indicates that the MP4 encoding is Advanced Video Coding Main Profile Level 4.2. Explaining this syntax is well beyond
+the scope of this lesson. Jake Archibald has a post explaining [how to determine the codec parameter for an AV1 video](https://jakearchibald.com/2022/html-codecs-parameter-for-av1/)
+if you want to learn more.
+
+When displaying a video, by default, the first frame of the video is shown as the still shot, or "poster", when it becomes available.
+This is something that can be controlled. The `poster` attribute enables the source of an image to show while the video is downloading
+and until it is played. Make sure to define the aspect ratio of your video, because when the video loads, a size difference
+between the poster and the video will cause a reflow and repaint.
+
+In case you were wondering, if the video plays and is subsequently paused, the poster is not re-shown.
+
```

Comment:
```suggestion
This is something that can be controlled. The `poster` attribute enables the source of an image to show while the video is downloading
and until it is played. If the video plays and is subsequently paused, the poster is not re-shown.

Make sure to define the aspect ratio of your video, because when the video loads, a size difference
between the poster and the video will cause a reflow and repaint.

```